browser/src/control/jsdialog/Util.StateChange.ts

```diff
@@ -21,7 +21,7 @@ function onStateChange(element: Element, callback: StateChangeCallback) {
 	var enabledCallback = function (mutations: Array<MutationRecord>) {
 		for (var i in mutations) {
 			if (mutations[i].attributeName === 'disabled') {
-				var enable = mutations[i].oldValue !== null;
+				var enable = mutations[i].oldValue !== null && mutations[i].oldValue !== '';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/MutationRecord/oldValue
`For attributes, it is the value of the changed attribute before the change.`

so if the `oldValue` was `''` (empty string) and it meant to be "disabled" -> now we want to enable widget
then the `enable` value should be `true` so we will do `removeAttribute('disabled')` inside synchronizeDisabledState

or I missed something?

Reply:
Then best would be to do `setAttribute('disabled', 'true')` not empty string for disabled case.
Possibly we should change all the places we do the same...

eg in edit we use true: https://github.com/CollaboraOnline/online/blob/9a9ff2663aefa5c794b39c111cf0d6293363f365/browser/src/control/Control.JSDialogBuilder.js#L1523
in unobutton not: https://github.com/CollaboraOnline/online/blob/9a9ff2663aefa5c794b39c111cf0d6293363f365/browser/src/control/Control.JSDialogBuilder.js#L2504

Reply:
That will be less confusing

Reply:
This approach is not making sense to me based on `oldValue`. In case of re-triggering same state, if previous state was set to disabled and we get message to disable it again, it will be enabled incorrectly. 

And so far I am not able to find the way to get the new value which we are trying to set in MutationObserver.

Reply:
```[MutationRecord.target](https://developer.mozilla.org/en-US/docs/Web/API/MutationRecord/target) Read only

    The node the mutation affected, depending on the MutationRecord.type.

        For attributes, it is the element whose attribute changed.```
        
isn't the node already changed when that code is called? it means you can check value of target